core/deployment/src/main/java/io/quarkus/deployment/steps/ConfigGenerationBuildStep.java

```diff
@@ -114,6 +115,12 @@ public class ConfigGenerationBuildStep {
             SmallRyeConfigBuilder.class, "withSources",
             SmallRyeConfigBuilder.class, ConfigSource[].class);
 
+    @BuildStep
+    void nativeSupport(BuildProducer<RuntimeInitializedClassBuildItem> runtimeInitializedClassProducer) {
+        runtimeInitializedClassProducer.produce(new RuntimeInitializedClassBuildItem(
+                "io.quarkus.runtime.configuration.RuntimeConfigBuilder$UuidConfigSource$Holder"));
+    }
```

Comment:
I'm not really satisfied with the class I put this in, so if you have any better suggestions @radcortez I'm all ears

Reply:
I think it is fine.